### ISSUE TITLE
#M13: Move resources to hackathon and/or allow for hackathon specific resources (and maybe participants to add resources)

### ISSUE THREAD

**Issue by stefdworschak** [Code-Institute-Community/ci-hackathon-app#125]

**Comment by TravelTimN** [comment]
Are you saying to move the "Useful Resources" over to the hackathon app?
Also, are you sure we want to have participants add resources? Any approval prior to displaying, though?

**Comment by stefdworschak** [comment]
@TravelTimN it's more of a conversation topic than a must do. I was thinking that we might want to have resources that should be only released to participants of a particular hackathon. For example, if it was a hackathon for another organisation for example we might want to release specific content or if it was a hackathon around a specific topic that would makes sense as well.

Letting participants add content was another thought. We could always add some sort of audit step in between as well.

What do you think overall about this?

**Comment by TravelTimN** [comment]
Yeah - all makes sense... just not sure how often or regular that'd be.
I can't think of too many viewable resources we have for previous hackathons that would be "temporary" or "only for cohort-X".
If you deem it something definitely worth adding, then grand, shouldn't be tough to implement!
But yes, audit control first would be very worthy.
